internal/signing/kms_signature_client.go

```diff
@@ -0,0 +1,138 @@
+package signing
```

Comment:
Wdyt about moving this to the `awskms` package so that we have things very well separated? Do you think that'd cause some import cycles?
Maybe we could have three subpackages in `signing`: `kms`, `env`, `channelaccount`

Reply:
Let's keep it simple. Having it separate in multiple packages would make it harder to find the implementations, having everything on the same package makes it easier to find and debug whenever necessary.